### PR TITLE
ML-01

### PR DESCRIPTION
**Pull request by izakian**
Hi there please take a look and fix bugs

### CODE REVIEW

**Review by hesam20052000 — CHANGES_REQUESTED** [review]
Overall is good but please do these changes first. Thanks

File: app.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from flask import Flask
 from random import randint
+import os
```
Comment (hesam20052000) [inline app.py]: Remove this

File: app.py [hunk 2]
```diff
@@ -12,10 +14,7 @@ def rand(low_num, high_num):
         int -- random in range
     """
     return randint(low_num, hight_num)
-
```
Comment (hesam20052000) [inline app.py:15]: Extra space

Comment (izakian) [reply app.py:15]: done

**Review by hesam20052000 — APPROVED** [review]
Thanks

**Pull request merged**